tracer/src/Datadog.Trace/LibDatadog/DataPipeline/TraceExporter.cs

```diff
@@ -7,16 +7,14 @@
 
 using System;
 using System.Collections.Generic;
-using System.Globalization;
-using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 using Datadog.Trace.Agent;
 using Datadog.Trace.Logging;
 using Datadog.Trace.Vendors.Newtonsoft.Json;
 using static Datadog.Trace.Agent.Api;
 
-namespace Datadog.Trace.LibDatadog;
+namespace Datadog.Trace.LibDatadog.DataPipeline;
```

Comment:
Should this namespace really be TraceExporter (or similar) seeing as it's called TraceExporter everywhere in the code. I worry the overloading here can only cause confusion (you have to know they're the same exact thing 😄)

Reply:
Started with TraceExporter namespace but settled on data pipeline since this is how it maps to Rust side.

It would be easier to reason about later https://github.com/DataDog/libdatadog/tree/eeecfbafc6e799122f67c506771aa20b720dbee2/data-pipeline

Reply:
In which case, I'd argue that we should rename all the TraceExporter types to be DataPipeline instead 😉 

Unless I'm conflating two things that are _actually_ different? Does the data-pipeline encapsulate anything _other_ than the trace exporter? Will it ever (from the managed side point of view)?

Reply:
it already does, CSS, dropping spans that are not required and some other things perhaps in future as we get more requirements.

currently we are handling telemetry as well with data pipeline which we shouldn't ideally.